homeassistant/components/mobile_app/entity.py

```diff
@@ -80,12 +81,12 @@ def icon(self):
         """Return the icon to use in the frontend, if any."""
         return self._config[ATTR_SENSOR_ICON]
 
-    @property
+    @cached_property
```

Comment:
The mobile app is only able to update these after the integration is set up. This caching means it will not be able to update it anymore.

Reply:
1e1d38c6f414c662750e3a8b767124b81a36e7d6